contracts/interchain-token-service/tests/execute.rs

```diff
@@ -58,27 +66,33 @@ fn register_update_its_contract_succeeds() {
 fn reregistering_same_chain_fails() {
     let mut deps = mock_dependencies();
     utils::instantiate_contract(deps.as_mut()).unwrap();
-
     let chain: ChainNameRaw = "ethereum".parse().unwrap();
     let address: Address = "0x1234567890123456789012345678901234567890"
         .parse()
         .unwrap();
+    let translation_contract: Address = MockApi::default()
+        .addr_make("translation_contract")
+        .to_string()
+        .parse()
+        .unwrap();
```

Comment:
use the new `address!` macro

Reply:
i don't think the macro works here because it needs a string literal? But we need an actual valid cosmos address, so it needs to be constructed via this addr_make call

Reply:
add a `cosmos_address!` macro instead that does this?

Reply:
created the new macro